rfcs/0027-trusted-bots.md

```diff
@@ -0,0 +1,75 @@
+---
+feature: trusted-bots
+start-date: 2018-03-23
+author: Matthew Bauer
+co-authors:
+related-issues: https://github.com/matthewbauer/rfcs/pull/27
+---
+
+# Summary
+[summary]: #summary
+
+Create a "trusted bot" process to manage GitHub bots. Bot maintainers can apply to make their bot "trusted", adding it to the bot team.
+
+# Motivation
+[motivation]: #motivation
+
+Why do we need bots? Nix has never had the accessibility of a package manager like Homebrew or the massive user base of a distro like Debian. To compete with them, we need to make our developers more productive. Bots can be a massive help.
+
+Currently running bots:
```

Comment:
What even is a bot? As far as I understand, Vulnix might not be strictly a bot — it seems to produce Markdown that is then published. And there are multiple package generators (* to Nix) that have the same mode of operation — generate content for a human to review and push.

On the other hand, https://monitor.nixos.org/ never did anything with the GitHub issues, but was constantly running, polling external source of information and providing its own web interface. Was it a bot?